pkg/distributor/distributor_test.go

```diff
@@ -1519,13 +1524,24 @@ func TestDistributor_Push_HistogramValidation(t *testing.T) {
 			errMsg: "received a sample whose timestamp is too far in the future",
 			errID:  globalerror.SampleTooFarInFuture,
 		},
+		"buckets at limit": {
```

Comment:
Should we add some variations of the test case where all of the buckets are negative or positive, or a mixture of positive and negative? (not sure whether this is best here or in the more detailed tests below in `validate_test.go`)

Reply:
I'll add in validate_test.go